ocean_provider/user_nonce.py

```diff
@@ -10,7 +10,11 @@ class UserNonce:
 
     def __init__(self, storage_path=None):
         self._storage_path = storage_path
-        self.storage = NonceStorage(storage_path)
+        NonceStorage(True, storage_path)
```

Comment:
can you please name this parameter explicitly? e.g. NonceStorage(storage_path, create_table=True)

Reply:
done